docker-compose.yml

```diff
@@ -13,7 +13,6 @@ services:
     - "3000:3000"
     - "3001:3001"
     volumes:
-    - Packages:/root/hac-website/Packages
```

Comment:
If we're not using this volume anymore, can we get rid of the definition of the volume in this file as well?